### ISSUE TITLE
__TRANSLATIONS_MODULE_STRING_TO_BE_REPLACED__

### ISSUE THREAD

**Issue by hazcod** [transferwise/webpack-translations-plugin#2]
Hi, i'm trying to get your plugin to work.
Version: `"webpack-translations-plugin": "^1.1.0"`

`webpack.config.js`:
```
  plugins: [
    new WebpackTranslationsPlugin({
      directory: 'translations',
      moduleName: 'translations',
      fileNameBase: 'data',
      development: IS_DEV,
    }),
```

directory tree:
```
ls -l
total 1144
drwx------    38   staff    1216 Aug 14 09:45 dist
drwx------  1048   staff   33536 Aug 14 09:20 node_modules
-rw-------     1   staff  563803 Aug 14 09:20 package-lock.json
-rwxr-xr-x     1   staff    3427 Aug 14 09:20 package.json
drwxr-xr-x@   36   staff    1152 Aug 14 09:19 src
drwx------     4   staff     128 Aug 14 09:38 translations
-rwxr-xr-x@    1   staff    5429 Aug 14 09:29 webpack.config.js
```

ls `translations/`:
```
data.en.json data.json
```

in `data.json` and `data.en.json`:
```
{
    "title": "Testing"
}
```

in my main js file:
```
import translations from 'translations';

const languages = Object.keys(translations);
console.log(languages);
console.log(translations);
```

But this returns:
```
Array(45) [ "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", … ]
main.077016c469de43c35fd2.js:1:17322
__TRANSLATIONS_MODULE_STRING_TO_BE_REPLACED__
```

Any ideas?

**Comment by hazcod** [comment]
Can confirm that this returns `undefined` aswel:
```
console.log(translations.languages);
console.log(translations.translations);
```

**Comment by hazcod** [comment]
CC @deakandris  @hendrikmolder  @lkokhreidze

**Comment by fergusjordantw** [comment]
This was broken when only one language was provided, but has been fixed in version 1.1.1
